.gitattributes

```diff
@@ -8,7 +8,7 @@
 .gitattributes           export-ignore
 .gitignore               export-ignore
 phpstan.neon             export-ignore
-phpunit.xml              export-ignore
+/phpunit.xml             export-ignore
```

Comment:
This is necessary otherwise the `phpunit.xml` stub is ignored on export. I guess we could call the stub `phpunit.xml.stub`, but that seems a bit weird when it's already in a `stubs` directory.

Reply:
No problem.